enterprise/utils.py

```diff
@@ -26,6 +29,11 @@
 except ImportError:
     Registry = None
 
+try:
+    from openedx.core.djangoapps.commerce.utils import ecommerce_api_client
+except ImportError:
+    ecommerce_api_client = None
```

Comment:
Instead of importing this from the edx-platform module, enterprise should have it's own ecommerce client defined, similar to how we have created clients for [discovery](https://github.com/edx/edx-enterprise/blob/master/enterprise/course_catalog_api.py) and [lms](https://github.com/edx/edx-enterprise/blob/master/enterprise/lms_api.py). It would also be nice to reorganize that API client code a bit, perhaps creating an `api_client` folder which could contain a file for each services' client code (lms, discovery, ecommerce).